source/includes/fundamentals/code-snippets/Geo.java

```diff
@@ -46,12 +46,19 @@ public void go() {
     private void nearExample() {
         // begin findExample
 
-        // code to set up your mongo client ...
+        // Add your MongoCollection setup code here
```

Comment:
Issue:
As the prior version of this comment mentions, this should be the "MongoClient" and not "MongoCollection".

Suggestion:
```suggestion
        // Add your MongoClient setup code here
```